Ginger/Ginger/Actions/_Common/ActUIElementLib/ActUIElementEditPage.xaml.cs

```diff
@@ -61,15 +61,36 @@ public ActUIElementEditPage(ActUIElement act)
             List<eLocateBy> LocateByList = mPlatform.GetPlatformUIElementLocatorsList();
             ElementLocateByComboBox.BindControl(mAction, nameof(ActUIElement.ElementLocateBy), LocateByList);
             ElementTypeComboBox.BindControl(mAction, nameof(ActUIElement.ElementType), mPlatform.GetPlatformUIElementsType());
+
+            //if widgets element, only supported to java platform now.
+            if (act.Platform.Equals(ePlatformType.Java))
+            {
+                ShowWidgetsElementCheckBox();
+            }
+
             SetLocateValueFrame();
             ShowPlatformSpecificPage();
             ShowControlSpecificPage();
             ElementLocateByComboBox.SelectionChanged += ElementLocateByComboBox_SelectionChanged;
         }
 
+        private void ShowWidgetsElementCheckBox()
+        {
+            xWidgetsElementType.Visibility = Visibility.Visible;
+            var isWidgetElement = mAction.GetOrCreateInputParam(Fields.WidgetsElement, "false");
```

Comment:
use binding to bind this check box with input value  instead of setting ischecked property explicitly